ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v1/page/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,80 @@
+                size="L">
+                <items jcr:primaryType="nt:unstructured">
+                    <commerce cq:showOnCreate="{Boolean}true"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Commerce"
+                        sling:orderBefore="cloudservices"
+                        sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
+                        <granite:rendercondition
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/renderconditions/or">
+                            <categoryPageCondition jcr:primaryType="nt:unstructured"
+                                sling:resourceType="core/cif/components/renderconditions/pagetemplate"
+                                templatePath="/conf/atest/settings/wcm/templates/category-page"/>
+                            <productPageCondition
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="core/cif/components/renderconditions/pagetemplate"
+                                templatePath="/conf/atest/settings/wcm/templates/product-page"/>
+                        </granite:rendercondition>
+                        <items jcr:primaryType="nt:unstructured">
+                            <column jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/container">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <magentoSection jcr:primaryType="nt:unstructured"
+                                        jcr:title="Commerce Settings"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <categoryFilter jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="commerce/gui/components/common/cifcategoryfield"
+                                                fieldDescription="Category ids for which this page will be used."
+                                                fieldLabel="Category ids for which this page will be used."
+                                                multiple="{Boolean}true"
+                                                name="./selectorFilter"
+                                                selectionId="id">
+                                                <granite:rendercondition jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="core/cif/components/renderconditions/pagetemplate"
+                                                    templatePath="/conf/atest/settings/wcm/templates/category-page"/>
```

Comment:
Who is `/conf/atest`?

Reply:
Since we need the `templatePath` here, which we don't have, should we move the portions or the entire dialog to the archetype?

Reply:
@dplaton , good catch!
@mhaack , I'm looking for an alternative since this dialog would normally belong to this project.

Reply:
I've eliminated the page template from the render conditions.

Reply:
LGTM now